### PR TITLE
Dynamic languages: Deprecate models-as-data CSV interface

### PR DESCRIPTION
**Pull request by hvitved**

### CODE REVIEW

**Review by yoff — DISMISSED** [review]
Python 👍 

Can we really get away with no change note, though? If people have models in CSV-files, they would need to convert them, right?

File: javascript/ql/lib/semmle/javascript/frameworks/data/internal/ApiGraphModels.qll [hunk 1]
```diff
@@ -134,15 +134,15 @@ module ModelInput {
      * `kind` should be either `value` or `taint`, for value-preserving or taint-preserving steps,
      * respectively.
      */
-    abstract predicate row(string row);
+    abstract deprecated predicate row(string row);
   }
 
   /**
    * A unit class for adding additional type model rows.
```
Comment (asgerf) [inline javascript/ql/lib/semmle/javascript/frameworks/data/internal/ApiGraphModels.qll:141]:
```suggestion
   * DEPRECATED. Use a data extension or the `TypeModel` class instead.
   *
   * A unit class for adding additional type model rows.
```

**Review by asgerf — DISMISSED** [review]
LGTM. I think the change note can be omitted, but perhaps the deprecated classes can have a QLDoc comment about how the contents should be migrated to a data extension instead. And in the case of `TypeModelCsv`, the `TypeModel` class is also an alternative.

**Review by yoff — APPROVED** [review]
LGTM

**Review by asgerf — APPROVED** [review]

**Pull request merged**